threadpoolctl.py

```diff
@@ -539,29 +137,36 @@ class threadpool_limits:
     the supported libraries to `limit`. This function works for libraries that
     are already loaded in the interpreter and can be changed dynamically.
 
-    The `limits` parameter can be either an integer or a dict to specify the
-    maximal number of thread that can be used in thread pools. If it is an
-    integer, sets the maximum number of thread to `limits` for each library
-    selected by `user_api`. If it is a dictionary `{{key: max_threads}}`, this
-    function sets a custom maximum number of thread for each `key` which can be
-    either a `user_api` or a `prefix` for a specific library. If None, this
-    function does not do anything.
-
-    The `user_api` parameter selects particular APIs of libraries to limit.
-    Used only if `limits` is an int. If it is None, this function will apply to
-    all supported libraries. If it is "blas", it will limit only BLAS supported
-    libraries and if it is "openmp", only OpenMP supported libraries will be
-    limited. Note that the latter can affect the number of threads used by the
-    BLAS libraries if they rely on OpenMP.
+    Parameters
+    ----------
+    limits : int, dict or None (default=None)
+        The maximal number of thread that can be used in thread pools
+
+        - If int, sets the maximum number of thread to `limits` for each
```

Comment:
I think we should use ["number of" + plural](https://ell.stackexchange.com/questions/87072/is-a-number-of-plural-or-singular). I know that this mistake was already present in the previous state of the code base, but we might as well fix it while we are at it.

```suggestion
        - If int, sets the maximum number of threads to `limits` for each
```